python/pyspark/pandas/tests/test_dataframe.py

```diff
@@ -5529,6 +5529,25 @@ def test_abs(self):
         self.assert_eq(abs(psdf), abs(pdf))
         self.assert_eq(np.abs(psdf), np.abs(pdf))
 
+    def test_corrwith(self):
+        df1 = ps.DataFrame({"A": [1, np.nan, 7, 8], "X": [5, 8, np.nan, 3], "C": [10, 4, 9, 3]})
+        df2 = df1[["A", "C"]]
+        self._test_corrwith(df1, df2)
+        self._test_corrwith(df1, df2.A)
+        self._test_corrwith(df1, df2.C)
```

Comment:
Can we add a simple chained-operation case ?? 

e.g. `self._test_corrwith((df1 + 1), df2.C)`